### ISSUE TITLE
Space before not nested selectors

### ISSUE THREAD

**Issue by movie4** [stylelint/stylelint#1020]
```
.header {
  width: 100%;
}
  .header__logo {
    padding: 0 0 0 35px;
  }
  .header__menu {
    margin: 20px 330px 0 0;
  }

```

Please help add right rule for pseudo nested blocks:

```
4:3	Expected empty line before non-nested rule (rule-non-nested-empty-line-before)
7:3	Expected empty line before non-nested rule (rule-non-nested-empty-line-before) 
```
Thanks

**Comment by jeddy3** [comment]
This rule is behaving correctly.

Your config appears to be:

```json
{
  "rule-non-nested-empty-line-before": "always"
}
```

And you have *no* empty lines between your *non-nested* rules:

```css
.header {
  width: 100%;
}
  .header__logo {
    padding: 0 0 0 35px;
  }
  .header__menu {
    margin: 20px 330px 0 0;
  }
```

Either change your config to:

```json
{
  "rule-non-nested-empty-line-before": "never"
}
```

*Or* add an empty line before each of your rules:


```css
.header {
  width: 100%;
}

  .header__logo {
    padding: 0 0 0 35px;
  }

  .header__menu {
    margin: 20px 330px 0 0;
  }
```

It is not possible to have a differing empty line style depending on the rule's selector and/or indentation. If you wish to have differing styles, either [turn the rule off](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/configuration.md#turning-rules-on-and-off) or write [plugin](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/plugins.md).

As a kind request, can you please use [stackoverflow](https://stackoverflow.com/questions/tagged/stylelint) for this type of support in the future.

**Comment by movie4** [comment]
Thanks for quick reply!
